lib/smoke_detector/middleware/javascript_monitors.rb

```diff
@@ -32,7 +34,7 @@ def monitoring_code
 
     def monitor?(headers)
       # minified code may not be UTF8 compliant, so you can't use present? reliably
-      headers["Content-Type"] =~ ACCEPTABLE_CONTENT && monitoring_code.size > 10
+      headers["Content-Type"] =~ ACCEPTABLE_CONTENT && monitoring_code.size > MINIMUM_SIZE_OF_JS
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.